2-mandatory/1-create-functions.js

```diff
@@ -43,9 +45,12 @@ Write a function that:
 - The numbers must be rounded to 2 decimal places.
 - Numbers greater 100 must be replaced with 100.
 */
+const formatPercentage = (arr) => arr.map(formatNumToStr);
```

Comment:
Again, really nice code Laleh! It works and is super readable! If you wanted to make this more succinct (a fancy word some coders use 😅) you could do this

`const formatPercentage = (arr) => arr.map(e => e >= 100 ? "100%" : `${Math.round(e * 100) / 100}%`);`

I believe we only learned about the `conditional / ternary operator` this week, but it's a great tool if we are using only one if statement, aka, if one thing is true, we only want to return a truth value and a false value. 

You can read up about them [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator).